client/src/app/components/Layout/Layout.jsx

```diff
@@ -41,8 +48,23 @@ export class Layout extends Component {
     loginUser();
   };
 
+  displayModeSwitch = () => {
+    const {
+      location: { pathname }
+    } = this.props;
+
+    return (
+      pathname.includes(`${Routes.COHORT}/`) ||
+      pathname.includes(Routes.DASHBOARD)
+    );
+  };
+
+  handleListModeChange = () =>
```

Comment:
when having an ENUM for this you will have two clear method 

switchToListViewType = () =>

switchToTileViewType = () =>